public/pages/Overview/components/Widgets/RecentThreatIntelFindingsWidget.tsx

```diff
@@ -92,14 +77,28 @@ export const RecentThreatIntelFindingsWidget: React.FC<RecentThreatIntelFindings
 
   return (
     <WidgetContainer title={'Recent threat intel findings'} actions={actions}>
-      <TableWidget
-        columns={columns}
-        items={findingItems}
-        sorting={{ sort: { field: 'timestamp', direction: SortDirection.DESC } }}
-        loading={loading}
-        message={widgetEmptyMessage}
-        className={widgetEmptyMessage ? 'sa-overview-widget-empty' : undefined}
-      />
+      {findingItems.length === 0 ? (
+        <EuiEmptyPrompt
```

Comment:
Nitpick: Looks like this empty prompt gets used in a few places. We could consider adding a constant for it in a helper file.

Reply:
+1, create a component out of it and use that in all places

Reply:
Created a helper function for the empty prompt so it's used across the widgets.